cli/src/util/spm.ts

```diff
@@ -82,3 +142,62 @@ let package = Package(
 
   return packageSwiftText;
 }
+
+export async function runCocoapodsDeintegrate(config: Config, options: MigrateSPMInteractiveOptions): Promise<void> {
+  const podPath = await config.ios.podPath;
```

Comment:
the function that configures the `podPath` already does all the checks for knowing if the user is using bundler or not, you shouldn't need to repeat all the checks

Reply:
Oh it does? I think this might be a copy and paste from another place that doesn't need it then. But that's an issue for another time.

Reply:
This had to stay as is, as the exec screws up otherwise.